index.js

```diff
@@ -18,6 +19,9 @@ module.exports = {
     "comment-whitespace-inside": "always",
     "declaration-bang-space-after": "never",
     "declaration-bang-space-before": "always",
+    "declaration-block-no-duplicate-properties": [ true, {
```

Comment:
It's part of the stylelint eslint - https://github.com/stylelint/eslint-config-stylelint/blob/master/eslintrc.json#L11-L13